app/models/path.rb

```diff
@@ -7,10 +7,22 @@ class Path < ActiveRecord::Base
   has_many :exercises, through: :guides
 
   def name
+
     if category.single_path?
       category.name
     else
       "#{category.name} (#{language.name})"
     end
+
+  end
+
+  def nameWithAllPaths #the ones that has guides attached and the ones that doesn't
```

Comment:
Please remember that in ruby we use snake case: `name_with_all_paths`. It is important to follow conventions in order to keep code consistent

On the other hand: this method is not really necessary - it is not used except for the test you fixed.   I would remove this method, and simply change the test - it is ok that the path name has changed, so you need to just change the asserted message